docs/navigation.yml

```diff
@@ -1,6 +1,5 @@
 nav:
-  - introduction
-  - brand
-  - components
+  - Overview
+  - Brand
+  - Components
```

Comment:
@ContentBrewmaster looks like these changes are the cause of a build error, from https://readthedocs.org/projects/civicactions-style-guide/builds/19332251/ is this error:

> mkdocs_awesome_pages_plugin.navigation.NavEntryNotFound: Nav entry "Overview" not found. [/home/docs/checkouts/readthedocs.org/user_builds/civicactions-style-guide/checkouts/latest/docs/navigation.yml]

Reply:
I changed it back to "Introduction" to see if it resolves.